test_conformance/math_brute_force/unary_two_results_float.cpp

```diff
@@ -124,8 +124,6 @@ static cl_int BuildKernelFn(cl_uint job_id, cl_uint thread_id UNUSED, void *p)
 
 int TestFunc_Float2_Float(const Func *f, MTdata d, bool relaxedMode)
 {
-    uint64_t i;
-    uint32_t j, k;
     uint32_t l;
```

Comment:
Is this `l` variable used anywhere? Can't see anything from a quick scan and wondering if it can be removed

Reply:
Indeed, it's not. There are a few others, like `testing_mode` a few lines below. I've actually got pending patches to address unused variables found by Clang's `-Wunused` warning flag.

Reply:
cool, should've guessed you'd have a pending patch for this 😄